src/applications/facility-locator/routes.jsx

```diff
@@ -2,18 +2,37 @@ import FacilityLocatorApp from './containers/FacilityLocatorApp';
 import FacilityDetail from './containers/FacilityDetail';
 import ProviderDetail from './containers/ProviderDetail';
 import VAMap from './containers/VAMap';
+import FacilitiesMap from './containers/FacilitesMap';
+import environment from 'platform/utilities/environment';
 
 const childRoutes = [
   { path: 'facility/:id', component: FacilityDetail },
   { path: 'provider/:id', component: ProviderDetail },
 ];
 
+const getMap = () => {
+  if (environment.isProduction()) return VAMap; // Use legacy map on prod
+
+  const isUnitTest = window.Mocha;
+  const isReviewEnvironment = environment.API_URL.includes('review.vetsgov');
+  const isLocal = environment.isLocalhost();
+  const isStaging = environment.isStaging();
+  const isCypress = window.Cypress;
+
+  if (isUnitTest || isCypress) return VAMap;
+
+  if (isReviewEnvironment || isLocal || isStaging) {
+    return FacilitiesMap;
+  }
+  return VAMap;
+};
```

Comment:
👍  - though at some point we will need to figure out how to write Cypress tests against the new map.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/14749, however, it should be merged after the map upgrade, it does not make any sense supporting test cases against 2 different map UI

Reply:
Have you tried running the current Cypress tests against the new Map?

Reply:
yes but needs fixes, that's why I have #14749.